.github/workflows/incubator-workflow.yml

```diff
@@ -0,0 +1,28 @@
+name: Build Incubator Workflow
+
+on:
+  push:
+    paths:
+      - 'debezium-core/**'
```

Comment:
`debezium-core` depends on `debezium-api` so it should be included here and in other instances too